http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/config/Config.scala

```diff
@@ -592,6 +592,17 @@ object Config {
     )
   }
 
+  implicit private val cromwellAppConfigReader: ValueReader[CromwellAppConfig] = ValueReader.relative { config =>
```

Comment:
would be nice if you could add it to [here](https://github.com/DataBiosphere/leonardo/blob/7e2f6d734d1205c723c47fa792ceaea1337caa75/http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/ConfigReader.scala#L21). It uses pureconfig, so that you don't have to manually write the config reader